config/api.yml

```diff
@@ -3992,6 +3992,12 @@
         :identifier:
         - vm_show
         - sui_vm_details_view
+      - :name: parent_choices
+        :identifier:
+        - we_don't_have_one_yet
+      - :name: child_choices
```

Comment:
this is not as mentioned ^^^^. Api returns now child_resources.

Reply:
do you mind telling me where that is?

Reply:
Oh _What I wrote here ?_ so again for certitude:

we don't need `child_choices` as I mentioned it above. Api returns `child_resources` at this moment - so it is ok. (we put [together table](https://github.com/ManageIQ/manageiq-api/issues/637#issuecomment-530361764) and we found that we need only parent_choices as you have it in this PR - thanks)

 it is `virtual_has_many` child_resources - [here](https://github.com/ManageIQ/manageiq/blob/a0bf167e134e46f2b7ff827c6dc71dc9a3c42c0d/app/models/vm_or_template.rb)

and in API request you need to expose it:
```
http://localhost:3090/api/vms/1178?attributes=child_resources
```